assignment3.rb

```diff
@@ -0,0 +1,55 @@
+  require 'csv'
+  require("./employee.rb")
+  require("./factory_method.rb")
+  require("./hr_class.rb")
```

Comment:
please change file name hr_class to hr and also for other files.

Reply:
changes done